app/models/search.rb

```diff
@@ -24,6 +24,7 @@ def results
     res = res.bot_author unless bot_author.nil?
     res = res.unlabelled unless unlabelled.nil?
     res = res.is_private(is_private) unless is_private.nil?
+    res = muted ? res.muted : res.unmuted
```

Comment:
Can we change this to act the same as `is_private` line above, so it only filters by muted if the muted: prefix is passed through